templates/partials/split_editor.haml

```diff
@@ -219,6 +219,10 @@
                 %input{ng-change:"updateCategory(rule)",ng-model:"rule.test._base",name:"days",number:"",ng-required:"rule.category._base",type:"text",class:"operand-small"}
                 -trans "days"
               .operation{ng-if:"rule._config.operands == 0"}
+
+              .operation{ng-if:'rule._config.type == "has_intent" || len(listBotsIntents) < 0'}
+                %select{ng-options:"botIntent as botIntent.name group by botIntent.bot_name for botIntent in listBotsIntents", ng-model:"rule.test._base.intent", ng-change:"updateCategory(rule)", ng-if:"listBotsIntents", style:"height:34px;width:159px", select2:""}
```

Comment:
We don't need of `ng-if:"listBotsIntents"` if the conditional above is correct, right?